distribution/lib/Standard/Base/0.0.0-dev/src/Data/Ordering/Comparator.enso

```diff
@@ -1,6 +1,6 @@
 from Standard.Base import all
 import Standard.Base.Data.Ordering.Natural_Order
-from Standard.Base.Data.Text.Text_Ordering as Text_Ordering_Module import Text_Ordering
+from Standard.Base.Data.Text.Text_Ordering import Text_Ordering
```

Comment:
I just wanted to say how awesome that change is! The need for this renaming in so many places was really quite annoying.